android/app/google-services.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Can you explain the contents of this file and why is it needed?

Reply:
I dont know what the contents of these file are for, but it was given by firebase when i registered the app. i believe its needed for the google sign in.